lib/index.js

```diff
@@ -62,9 +62,8 @@ internals.implementation = function (server, options) {
 
       getKey(request, token, function(err, key, extraInfo){
         if (err) { return reply(Boom.wrap(err)); }
-
         // handle err
-        jwt.verify(token, key, function(err, decoded) {
+        jwt.verify(token, key, settings, function(err, decoded) {
```

Comment:
considering we are adding two more settings, could you please add some validation for the `options` object using `Joi`?

We should have done this before, but this is a good opportunity to get some validation and tests for it.

Thanks,